ContentView.swift

```diff
@@ -0,0 +1,308 @@
+                        Image(systemName: "forward.fill")
+                            .font(size.height < 300 ? .title3 : .title)
+                            .onTapGesture {
+                                playNextSong()
+                            }
+                    })
+                }
+                .foregroundStyle(Color.white)
+                .frame(maxWidth: .infinity)
+                
+                VStack(spacing: spacing) {
+                    HStack(spacing: 15) {
+                        Image(systemName: "speaker.fill")
+                        
+                        Capsule()
+                            .fill(.ultraThinMaterial)
+                            .environment(\.colorScheme, .light)
+                            .frame(height: 5)
+                        Image(systemName: "speaker.wave.3.fill")
+                    }
+                    HStack(alignment: .top, spacing: size.width * 0.18, content: {
+                        Button(action: {
+                            
+                        }, label: {
+                            Image(systemName: "quote.bubble")
+                                .font(.title2)
+                        })
+                        
+                        VStack(spacing: 6, content: {
+                            Button(action: {
+                                
+                            }, label: {
+                                Image(systemName: "airpodspro.chargingcase.wireless.fill")
+                                    .font(.title2)
+                            })
+                            Text("Hi")
+                            Text("Airpods")
+                                .font(.caption)
+                                .foregroundStyle(Color.white)
+                        })
+                        
+                        Button(action: {
+                            
+                        }, label: {
+                            Image(systemName: "list.bullet")
+                                .font(.title2)
+                        })
+                        
+                        
+                    })
+                    .foregroundStyle(Color.white)
+                    .padding(.top, spacing)
+                    .opacity(0.2)
+                }
+            }
+        }
+    }
+}
+
+struct ContentView_Preview: PreviewProvider {
+    static var previews: some View {
+        ContentView(expandSheet: .constant(true), animation: Namespace().wrappedValue)
+            .preferredColorScheme(.dark)
+    }
+}
+
+extension View {
+    var deviceCornerRadius: CGFloat {
+        
+        let key = "_displayCornerRadius"
+        
+        if let screen = (UIApplication.shared.connectedScenes.first as? UIWindowScene)?.windows.first?.screen{
+            if let cornerRadius = screen.value(forKey: key) as? CGFloat {
+                return cornerRadius
+            }
+            return 0
+        }
+        return 0
+    }
```

Comment:
Nested If inside If is not elegant, try to merge conditions if you expect the same result in case where condition is not TRUE
```suggestion
        var deviceCornerRadius: CGFloat {
        guard let screen = (UIApplication.shared.connectedScenes.first as? UIWindowScene)?.windows.first?.screen,
              let cornerRadius = screen.value(forKey: "_displayCornerRadius") as? CGFloat
        else {
            return 0
        }
        return cornerRadius
    }
```